hazelcast/src/main/resources/hazelcast-config-5.2.json

```diff
@@ -4223,6 +4226,15 @@
         }
       }
     },
+    "SqlClient": {
```

Comment:
Can we add a test for this, as described in https://github.com/hazelcast/hazelcast/wiki/Checklist-for-Hazelcast-config-changes#yaml-configuration-schema-update

Reply:
Done